dist/images/ovnkube.sh

```diff
@@ -665,10 +665,6 @@ ovn-master () {
   wait_for_event ready_to_start_node
   echo "ovn_nbdb ${ovn_nbdb}   ovn_sbdb ${ovn_sbdb}"
 
-  # wait for northd to start
```

Comment:
Hmm, why doesn't master need to wait for northd? In the containerized case we want to do so because the OVN container would be starting concurrently with the ovnkube one. Or perhaps I misunderstand.

Reply:
@dcbw there is nothing in `ovnkube --master` container that depends on `ovn-northd` container being up and running. They can come up parallely or in any order.

This commit is in preparation for running our Pods with HostPID set to False. I am trying to remove all these checks where one container checks the PID of the other container when it is not required.

Reply:
@girishmg At some point along the way I needed that to get ovn to come up properly. There are/were a lot of races and some of them didn't make much sense. It may no longer be needed but I would test it a lot. Maybe put a 30 sec sleep before starting it and make sure the cluster can start pods properly. If northd is required for normal operation you need make sure its there before starting normal operation. ovnkube --master is the last thing to come up and it says we are ready to go.

Reply:
@pecameron good point. Will test with your suggestion and get back. Thanks.

Reply:
@girishmg fair enough on northd. Does the readiness check you added cover northd then? eg we should make sure that if northd is running as a container that the entire pod is not "ready" until northd is ready and running.

Reply:
@dcbw yes we have checks in place to ensure that. But, I stumbled upon an issue (something to do with onkube --node containers) that will make this change infeasible. See my comment below...